example/assets/tests/buttons.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "Buttons",
-  "steps": [{
+  "steps": [
```

Comment:
Rather than reusing (and changing) this test, please create a net-new test as well as an example page for the test.

For reference, that's the direction I'm heading to showcase steps in a more atomic way.  The current tests exist to help me test things that were problem areas within the framework myself while I was developing it.